gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
Left it in because I don't want to assume what machine these are being run on e.g. clients at Jaguar used windows so this would be beneficial for them